server/server.go

```diff
@@ -202,7 +203,7 @@ func (s *Server) restore(ctx context.Context) []string {
 		}
 		log.Warnf(ctx, "could not restore sandbox %s container %s: %v", metadata.PodID, sbID, err)
 		for _, n := range names[sbID] {
-			if err := s.Store().DeleteContainer(n); err != nil {
+			if err := s.Store().DeleteContainer(n); err != nil && err != storageTypes.ErrNotAContainer {
```

Comment:
In what situations would this error be returned by storage?

Reply:
The names stored for a container contain both the container ID and the container name. This comes up when we attempt to delete the container's name